Cleanup filenames for export_to_cluster

lparsons opened the pull request:
Remove problematic characters from filenames and add file extension.

lparsons committed:
Cleanup filenames for export_to_cluster

nsoranzo commented on tools/export_to_cluster/export_to_cluster.py:
```diff
@@ -32,8 +33,14 @@
+        print("'%s' copied" % dn_safe)
     except Exception as e:
-        print("Error copying '%s', %s" % (dn, e))
+    sys.exit("Error copying '%s', %s" % (dn_safe, e))
```
`sys.exit()` will stop the loop, the use of `print()` was intentional. Also this is not correctly indented.

nsoranzo commented on tools/export_to_cluster/export_to_cluster.py:
```diff
@@ -6,6 +6,7 @@
 import os.path
 import shutil
 import sys
+import re 
```
Can you remove the trailing whitespace to make flake8 happy?

nsoranzo commented on tools/export_to_cluster/export_to_cluster.xml:
```diff
@@ -6,7 +6,7 @@
 python $__tool_directory__/export_to_cluster.py -d '$export_dir' -p "\$EXPORT_DIR_PREFIX"
 #for $d in $datasets
-    '$d' '$d.hid: $d.name'
+    '$d' '$d.hid: $d.name.$d.ext'
```
`${d.name}.${d.ext}` is probably clearer.

lparsons committed:
Continue on copy error, improve messaging

Continue looping over files if one errors, output the error to the log
(stdout) as well as stderr so the tool errors to indicate to the user
something went wrong.

Cleanup some code style issues.

nsoranzo commented:
Fantastic, thanks @lparsons!

nsoranzo commented:
@lparsons The tool has been pushed to the MTS.
